samples/README.md

```diff
@@ -0,0 +1,2 @@
+**Samples**
+This folder contains a directory for each parallel pattern that includes the examples related to them.
```

Comment:
This directory contains a set of samples programs for *GrPPI*.

Currently you will find:
* **map**: Examples for the map pattern.